docs/recipes/browser-testing.md

```diff
@@ -7,29 +7,33 @@ An example of this is React, at least if you want to use ReactDOM.render and sim
 
 This recipe works for any library that needs a mocked browser environment.
 
-## Install jsdom
+## Install node-browser-environment
 
-Install [jsdom](https://github.com/tmpvar/jsdom).
+Install [node-browser-environment](https://github.com/lukechilds/node-browser-environment).
 
-> A JavaScript implementation of the WHATWG DOM and HTML standards, for use with node.js
+> Simulates a global browser environment using jsdom.
 
 ```
-$ npm install --save-dev jsdom
+$ npm install --save-dev node-browser-environment
```

Comment:
@lukechilds Not very relevant in this situation, as it's a top-level module, so maybe not necessary, but when your module is deep in a dependency tree, people will start opening issues on every module that depends on it somewhere in the dependency tree about updating, which is very annoying for the maintainers and a lot of wasted time. Users think warnings are critical.